### ISSUE TITLE
Defunct?  Simple NN with gradient descent back propogation.

### ISSUE THREAD

**Issue by uDude** [lfex/lsci#41]
Is this project defunct?

I've never called another beam language from elixir (except erlang). 

 I'd like to write this simple example (logistic regression neural net with gradient descent back propogation) if this project is alive; otherwise, I'll just write it in erlang and happily contribute an LFE example.

I need the concurrency of erts with GPU computations.
